CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contribution guide
+
+Simple-Chat is an OpenSource Angular 4 Based Chatting Application. You can Contribute to this by
+
+  * Adding new Features.
+  * Reporting Bugs
+  * Suggesting Enhancements
+  * Your First Code Contribution
+  * Pull Requests
+  
+# How to Contribute.
+
+1. [Fork](https://github.com/mbrandau/simple-chat/fork) the repository
+2. Run npm install && cd frontend && npm install to install all required packages.
+
+## Did you find a bug?
+
+Ensure the bug was not already reported by searching on GitHub under Issues.
+
+If you're unable to find an open issue addressing the problem, open a new one. Be sure to include a title and clear description, as much relevant information as possible and 
```

Comment:
This section is good :+1: we'll include this in the contribution guide. I'd like to stick to #19. Once it's merged, feel free to add this section here.

Reply:
Ok :)

Reply:
Can you move this above the "Getting started" modify the getting started section so that we don't mention things twice?
And by the way you're just ending this sentence with "as possible and" :smile:

Reply:
Sure :)